### ISSUE TITLE
Could not create user inside installation

### ISSUE THREAD

**Issue by tyler820201** [archlinux/archinstall#2416]
Hardware model detected: Notebook NL5xRU; UEFI mode: True
Processor model detected: AMD Ryzen 7 4700U with Radeon Graphics
Memory statistics: 63397544 available out of 65199472 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

Virtualization detected: None; is VM: False
Graphics devices detected: dict_keys(['Advanced Micro Devices, Inc. [AMD/ATI] Renoir [Radeon RX Vega 6 (Ryzen 4000/5000 Mobile Series)] (rev c2)'])
Disk states before installing: [
    {
        "children": [],
        "fsavail": 0,
        "fsroots": [
            "."
        ],
        "fstype": "squashfs",
        "fsuse_percentage": "100%",
        "fsver": "4.0",
        "log_sec": 512,
        "mountpoint": "/run/archiso/airootfs",
        "mountpoints": [
            "/run/archiso/airootfs"
        ],
        "name": "loop0",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/loop0",
        "pkname": null,
        "pttype": null,
        "ptuuid": null,
        "rota": false,
        "size": "776 MiB",
        "tran": null,
        "type": "loop",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ntfs",
                "fsuse_percentage": null,
                "fsver": null,
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sda1",
                "partn": 1,
                "parttype": "ebd0a0a2-b9e5-4433-87c0-68b6b72699c7",
                "partuuid": "94966622-1271-4e07-906d-4d753caa399c",
                "path": "/dev/sda1",
                "pkname": "sda",
                "pttype": "gpt",
                "ptuuid": "7868ff30-8fa8-427c-9856-92bb1ba4b748",
                "rota": false,
                "size": "3440970 MiB",
                "tran": null,
                "type": "part",
                "uuid": "01CDEADADB804880"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sda",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sda",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "7868ff30-8fa8-427c-9856-92bb1ba4b748",
        "rota": false,
        "size": "3815447 MiB",
        "tran": "sata",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [
                    {
                        "children": [],
                        "fsavail": null,
                        "fsroots": [],
                        "fstype": "iso9660",
                        "fsuse_percentage": null,
                        "fsver": "Joliet Extension",
                        "log_sec": 512,
                        "mountpoint": null,
                        "mountpoints": [],
                        "name": "ventoy",
                        "partn": null,
                        "parttype": null,
                        "partuuid": null,
                        "path": "/dev/mapper/ventoy",
                        "pkname": "sdb1",
                        "pttype": "dos",
                        "ptuuid": "8cc6f18c",
                        "rota": true,
                        "size": "942 MiB",
                        "tran": null,
                        "type": "dm",
                        "uuid": "2024-03-01-15-06-37-00"
                    }
                ],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ntfs",
                "fsuse_percentage": null,
                "fsver": null,
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdb1",
                "partn": 1,
                "parttype": "0x7",
                "partuuid": "7e46bc40-01",
                "path": "/dev/sdb1",
                "pkname": "sdb",
                "pttype": "dos",
                "ptuuid": "7e46bc40",
                "rota": true,
                "size": "59359 MiB",
                "tran": null,
                "type": "part",
                "uuid": "209B0CF40E39BB02"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT16",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdb2",
                "partn": 2,
                "parttype": "0xef",
                "partuuid": "7e46bc40-02",
                "path": "/dev/sdb2",
                "pkname": "sdb",
                "pttype": "dos",
                "ptuuid": "7e46bc40",
                "rota": true,
                "size": "32 MiB",
                "tran": null,
                "type": "part",
                "uuid": "79CB-53D7"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sdb",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sdb",
        "pkname": null,
        "pttype": "dos",
        "ptuuid": "7e46bc40",
        "rota": true,
        "size": "59392 MiB",
        "tran": "usb",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT32",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p1",
                "partn": 1,
                "parttype": "ebd0a0a2-b9e5-4433-87c0-68b6b72699c7",
                "partuuid": "00b65700-1a47-43f2-b092-be6f436d67a8",
                "path": "/dev/nvme0n1p1",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "1086 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "F8B2-360B"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p2",
                "partn": 2,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "a2fc3e4f-c11d-44d0-9e4b-e2783d57239d",
                "path": "/dev/nvme0n1p2",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "151321 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "0c26d5d7-3f0e-467c-9044-4142f560492a"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p3",
                "partn": 3,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "cbbdb71d-8a2d-46ad-b2f3-be10c690eaca",
                "path": "/dev/nvme0n1p3",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "154500 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "2b0815ad-9b97-4759-9d3d-398f8761092c"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p4",
                "partn": 4,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "aede83dd-387d-4804-b480-0f0dca934882",
                "path": "/dev/nvme0n1p4",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "104400 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "3198a453-ff29-43e0-8ac8-264ad0ebc2bf"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "nvme0n1",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/nvme0n1",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
        "rota": false,
        "size": "476940 MiB",
        "tran": "nvme",
        "type": "disk",
        "uuid": null
    }
]
Checking version...
Failed determine pacman version: ['/usr/bin/pacman', '-Qu', 'archinstall'] exited with abnormal exit code [1]: [?25l[?25h
Hardware model detected: Notebook NL5xRU; UEFI mode: True
Processor model detected: AMD Ryzen 7 4700U with Radeon Graphics
Memory statistics: 63403496 available out of 65199472 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

Virtualization detected: None; is VM: False
Graphics devices detected: dict_keys(['Advanced Micro Devices, Inc. [AMD/ATI] Renoir [Radeon RX Vega 6 (Ryzen 4000/5000 Mobile Series)] (rev c2)'])
Disk states before installing: [
    {
        "children": [],
        "fsavail": 0,
        "fsroots": [
            "."
        ],
        "fstype": "squashfs",
        "fsuse_percentage": "100%",
        "fsver": "4.0",
        "log_sec": 512,
        "mountpoint": "/run/archiso/airootfs",
        "mountpoints": [
            "/run/archiso/airootfs"
        ],
        "name": "loop0",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/loop0",
        "pkname": null,
        "pttype": null,
        "ptuuid": null,
        "rota": false,
        "size": "776 MiB",
        "tran": null,
        "type": "loop",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ntfs",
                "fsuse_percentage": null,
                "fsver": null,
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sda1",
                "partn": 1,
                "parttype": "ebd0a0a2-b9e5-4433-87c0-68b6b72699c7",
                "partuuid": "94966622-1271-4e07-906d-4d753caa399c",
                "path": "/dev/sda1",
                "pkname": "sda",
                "pttype": "gpt",
                "ptuuid": "7868ff30-8fa8-427c-9856-92bb1ba4b748",
                "rota": false,
                "size": "3440970 MiB",
                "tran": null,
                "type": "part",
                "uuid": "01CDEADADB804880"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sda",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sda",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "7868ff30-8fa8-427c-9856-92bb1ba4b748",
        "rota": false,
        "size": "3815447 MiB",
        "tran": "sata",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [
                    {
                        "children": [],
                        "fsavail": null,
                        "fsroots": [],
                        "fstype": "iso9660",
                        "fsuse_percentage": null,
                        "fsver": "Joliet Extension",
                        "log_sec": 512,
                        "mountpoint": null,
                        "mountpoints": [],
                        "name": "ventoy",
                        "partn": null,
                        "parttype": null,
                        "partuuid": null,
                        "path": "/dev/mapper/ventoy",
                        "pkname": "sdb1",
                        "pttype": "dos",
                        "ptuuid": "8cc6f18c",
                        "rota": true,
                        "size": "942 MiB",
                        "tran": null,
                        "type": "dm",
                        "uuid": "2024-03-01-15-06-37-00"
                    }
                ],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ntfs",
                "fsuse_percentage": null,
                "fsver": null,
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdb1",
                "partn": 1,
                "parttype": "0x7",
                "partuuid": "7e46bc40-01",
                "path": "/dev/sdb1",
                "pkname": "sdb",
                "pttype": "dos",
                "ptuuid": "7e46bc40",
                "rota": true,
                "size": "59359 MiB",
                "tran": null,
                "type": "part",
                "uuid": "209B0CF40E39BB02"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT16",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdb2",
                "partn": 2,
                "parttype": "0xef",
                "partuuid": "7e46bc40-02",
                "path": "/dev/sdb2",
                "pkname": "sdb",
                "pttype": "dos",
                "ptuuid": "7e46bc40",
                "rota": true,
                "size": "32 MiB",
                "tran": null,
                "type": "part",
                "uuid": "79CB-53D7"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sdb",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sdb",
        "pkname": null,
        "pttype": "dos",
        "ptuuid": "7e46bc40",
        "rota": true,
        "size": "59392 MiB",
        "tran": "usb",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT32",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdc1",
                "partn": 1,
                "parttype": "c12a7328-f81f-11d2-ba4b-00a0c93ec93b",
                "partuuid": "0b2008ab-3d4b-44c7-8b1e-5ab2e155c6b8",
                "path": "/dev/sdc1",
                "pkname": "sdc",
                "pttype": "gpt",
                "ptuuid": "a5452e0c-4513-4705-8ecf-eb2a7a9aaeb9",
                "rota": true,
                "size": "511 MiB",
                "tran": null,
                "type": "part",
                "uuid": "F6C5-03E9"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sdc2",
                "partn": 2,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "48952dec-7cb7-4393-bd66-df06195158c9",
                "path": "/dev/sdc2",
                "pkname": "sdc",
                "pttype": "gpt",
                "ptuuid": "a5452e0c-4513-4705-8ecf-eb2a7a9aaeb9",
                "rota": true,
                "size": "714891 MiB",
                "tran": null,
                "type": "part",
                "uuid": "90166494-40cf-405e-914a-f3cb72404f4d"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sdc",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sdc",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "a5452e0c-4513-4705-8ecf-eb2a7a9aaeb9",
        "rota": true,
        "size": "715404 MiB",
        "tran": "usb",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT32",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p1",
                "partn": 1,
                "parttype": "ebd0a0a2-b9e5-4433-87c0-68b6b72699c7",
                "partuuid": "00b65700-1a47-43f2-b092-be6f436d67a8",
                "path": "/dev/nvme0n1p1",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "1086 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "F8B2-360B"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p2",
                "partn": 2,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "a2fc3e4f-c11d-44d0-9e4b-e2783d57239d",
                "path": "/dev/nvme0n1p2",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "151321 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "0c26d5d7-3f0e-467c-9044-4142f560492a"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p3",
                "partn": 3,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "cbbdb71d-8a2d-46ad-b2f3-be10c690eaca",
                "path": "/dev/nvme0n1p3",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "154500 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "2b0815ad-9b97-4759-9d3d-398f8761092c"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ext4",
                "fsuse_percentage": null,
                "fsver": "1.0",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme0n1p4",
                "partn": 4,
                "parttype": "0fc63daf-8483-4772-8e79-3d69d8477de4",
                "partuuid": "aede83dd-387d-4804-b480-0f0dca934882",
                "path": "/dev/nvme0n1p4",
                "pkname": "nvme0n1",
                "pttype": "gpt",
                "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
                "rota": false,
                "size": "104400 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "3198a453-ff29-43e0-8ac8-264ad0ebc2bf"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "nvme0n1",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/nvme0n1",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "0ce68eb9-40c5-40b7-b8a6-1a6bd0ff9129",
        "rota": false,
        "size": "476940 MiB",
        "tran": "nvme",
        "type": "disk",
        "uuid": null
    }
]
Checking version...
Failed determine pacman version: ['/usr/bin/pacman', '-Qu', 'archinstall'] exited with abnormal exit code [1]: [?25l[?25h
Password strength: very weak
Password strength: very weak
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/__init__.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/custom.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktop.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/minimal.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/server.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/tailored.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/xorg.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/applications/__init__.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/applications/pipewire.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/__init__.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/awesome.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/bspwm.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/budgie.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/cinnamon.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/cutefish.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/deepin.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/enlightenment.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/gnome.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/hyprland.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/i3.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/kde.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/lxqt.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/mate.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/qtile.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/sway.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/desktops/xfce4.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/__init__.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/cockpit.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/docker.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/httpd.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/lighttpd.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/mariadb.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/nginx.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/postgresql.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/sshd.py
Importing profile: /usr/lib/python3.11/site-packages/archinstall/default_profiles/servers/tomcat.py
 -- Chosen configuration --
{
    "__separator__": null,
    "additional-repositories": [],
    "archinstall-language": "English",
    "audio_config": {
        "audio": "pipewire"
    },
    "bootloader": "Grub",
    "config_version": "2.7.2",
    "debug": false,
    "disk_config": {
        "config_type": "manual_partitioning",
        "device_modifications": [
            {
                "device": "/dev/sdc",
                "partitions": [
                    {
                        "btrfs": [],
                        "dev_path": "/dev/sdc1",
                        "flags": [
                            "Boot",
                            "ESP"
                        ],
                        "fs_type": "fat32",
                        "mount_options": [],
                        "mountpoint": "/boot",
                        "obj_id": "ccaa7bac-c095-4405-a1fe-b1feaa89153a",
                        "size": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "B",
                            "value": 535822336
                        },
                        "start": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "sectors",
                            "value": 2048
                        },
                        "status": "existing",
                        "type": "primary"
                    },
                    {
                        "btrfs": [],
                        "dev_path": "/dev/sdc2",
                        "flags": [],
                        "fs_type": "ext4",
                        "mount_options": [],
                        "mountpoint": "/",
                        "obj_id": "54434a5b-3ddd-4b6b-bf22-13c4a97f39a6",
                        "size": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "B",
                            "value": 749617545216
                        },
                        "start": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "sectors",
                            "value": 1050624
                        },
                        "status": "existing",
                        "type": "primary"
                    }
                ],
                "wipe": false
            }
        ]
    },
    "disk_encryption": null,
    "hostname": "archlinux",
    "kernels": [
        "linux",
        "linux-lts",
        "linux-zen"
    ],
    "locale_config": {
        "kb_layout": "us",
        "sys_enc": "UTF-8",
        "sys_lang": "en_US"
    },
    "mirror_config": null,
    "network_config": {
        "type": "nm"
    },
    "no_pkg_lookups": false,
    "ntp": true,
    "offline": false,
    "packages": [],
    "parallel downloads": 0,
    "profile_config": {
        "gfx_driver": "AMD / ATI (open-source)",
        "greeter": "sddm",
        "profile": {
            "custom_settings": {
                "Kde": {}
            },
            "details": [
                "Kde"
            ],
            "main": "Desktop"
        }
    },
    "save_config": null,
    "script": "guided",
    "silent": false,
    "skip_ntp": false,
    "skip_version_check": false,
    "swap": false,
    "timezone": "Europe/London",
    "uki": false,
    "version": "2.7.2"
}
Unmounting all existing partitions: /dev/sdc
Unmounting: /dev/sdc1
Unmounting: /dev/sdc2
Use existing device: /dev/sdc
Creating partitions: /dev/sdc
Unmounting all existing partitions: /dev/sdc
Unmounting: /dev/sdc1
Unmounting: /dev/sdc2
Starting installation
Mounting partitions in order
Mounting /dev/sdc2: mount /dev/sdc2 /mnt/archinstall
Mounting /dev/sdc1: mount /dev/sdc1 /mnt/archinstall/boot
Waiting for time sync (timedatectl show) to complete.
Waiting for automatic mirror selection (reflector) to complete.
Waiting for Arch Linux keyring sync (archlinux-keyring-wkd-sync) to complete.
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

The multilib flag is not set. This system will be installed without multilib repositories enabled.
The testing flag is not set. This system will be installed without testing repositories enabled.
Installing packages: ['base', 'base-devel', 'linux-firmware', 'linux', 'linux-lts', 'linux-zen', 'amd-ucode']
Enabling periodic TRIM
Enabling service fstrim.timer
Installing packages: ['grub']
Adding bootloader Grub to /dev/sdc1
Installing packages: ['grub']
kernel parameters: rootfstype=ext4
GRUB boot partition: /dev/sdc1
GRUB EFI partition: /dev/sdc1
Installing packages: ['efibootmgr']
Installing packages: ['networkmanager']
Installing packages: ['network-manager-applet']
Enabling service NetworkManager.service
Creating user tyler
Could not create user inside installation: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'useradd', '-m', '-G', 'wheel', 'tyler'] exited with abnormal exit code [9]: useradd: user 'tyler' already exists

Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/archinstall/lib/installer.py", line 1301, in user_create
    SysCommand(f'/usr/bin/arch-chroot {self.target} useradd -m -G wheel {user}')
  File "/usr/lib/python3.11/site-packages/archinstall/lib/general.py", line 365, in __init__
    self.create_session()
  File "/usr/lib/python3.11/site-packages/archinstall/lib/general.py", line 414, in create_session
    with SysCommandWorker(
  File "/usr/lib/python3.11/site-packages/archinstall/lib/general.py", line 197, in __exit__
    raise SysCallError(
archinstall.lib.exceptions.SysCallError: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'useradd', '-m', '-G', 'wheel', 'tyler'] exited with abnormal exit code [9]: useradd: user 'tyler' already exists

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/archinstall/__init__.py", line 332, in run_as_a_module
    main()
  File "/usr/lib/python3.11/site-packages/archinstall/__init__.py", line 327, in main
    importlib.import_module(mod_name)
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 256, in <module>
    perform_installation(archinstall.storage.get('MOUNT_POINT', Path('/mnt')))
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 118, in perform_installation
    with Installer(
  File "/usr/lib/python3.11/site-packages/archinstall/lib/installer.py", line 109, in __exit__
    raise exc_val
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 175, in perform_installation
    installation.create_users(users)
  File "/usr/lib/python3.11/site-packages/archinstall/lib/installer.py", line 1283, in create_users
    self.user_create(user.username, user.password, user.groups, user.sudo)
  File "/usr/lib/python3.11/site-packages/archinstall/lib/installer.py", line 1303, in user_create
    raise SystemError(f"Could not create user inside installation: {err}")
SystemError: Could not create user inside installation: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'useradd', '-m', '-G', 'wheel', 'tyler'] exited with abnormal exit code [9]: useradd: user 'tyler' already exists

Archinstall experienced the above error. If you think this is a bug, please report it to
https://github.com/archlinux/archinstall and include the log file "/var/log/archinstall/install.log".

Hint: To extract the log from a live ISO 
curl -F'file=@/var/log/archinstall/install.log' https://0x0.st

**Comment by tyler820201** [comment]
[install.log](https://github.com/archlinux/archinstall/files/14580670/install.log)

**Comment by lizhipengpeng** [comment]
same error, how to solve it?

**Comment by svartkanin** [comment]
According to the configuration you're installation a system to existing partitions, I assume you have 1 boot partition and a 1 single root partition setup. If that one contains a installation already with the user "tyler" than that will most likely clash.

**Comment by tyler820201** [comment]
> According to the configuration you're installation a system to existing partitions, I assume you have 1 boot partition and a 1 single root partition setup. If that one contains a installation already with the user "tyler" than that will most likely clash.

Thanks. You are right. I did not wipe the existing install on the creating partitions section. I always use manuall installation but from the previous installation attempt it looks like i did not wipe the / partition.

**Comment by tyler820201** [comment]
> same error, how to solve it?

Yep. Just make sure u are wiping the partition properly before installation. I did try multiple time archinstall with different configurations and i did not wipe/format the system partition from the previous install.

**Comment by tyler820201** [comment]
I think i can close this topic.
By the way the install script still failed but that is related to an other issue. The script will fail if you want KDE Plasma as DE. The problem is that some plasma related package name has been changed and therefore unable tu download and install those packages... I dont know is there already an open ticket for this issue or not....  
I just add the problematic packages to additional packages to download with their new name. And that works. 🤓 Even if the installscript will finish with error i can reboot into tty and enable sddm and then reboot into the fresh arch install with KDE Plasma6. 😉 The problem is that plasma6 has  changed the package name for some packages. For example plasma-wayland-session is plasma-wayland-protocols.
I did just try the february and march iso but new release will probably fix it.
